client/src/pages/demo/components/index.tsx

```diff
@@ -32,6 +33,19 @@ const DemoComponent: React.FunctionComponent = () => {
         <br />
         <hr className="w-3/4 mx-auto" />
       </div>
+
+      <div className="p-4 border -b border-black">
+        <Card>
+          <main>Content goes here</main>
+        </Card>
+        <div className="mt-[5px]">
+          <h1>Props: Card</h1>
```

Comment:
- should be an anchor tag. please follow here https://github.com/framgia/sph-lms/pull/10#discussion_r1123993692